collect_app/src/main/AndroidManifest.xml

```diff
@@ -78,7 +78,8 @@ the specific language governing permissions and limitations under the License.
         android:largeHeap="true"
         android:networkSecurityConfig="@xml/network_security_config"
         android:supportsRtl="true"
-        android:usesCleartextTraffic="true">
+        android:usesCleartextTraffic="true"
+        android:theme="@style/Theme.Collect">
```

Comment:
If we have it here we probably do not need it in geo activities?:
```
        <activity
            android:name="org.odk.collect.geo.GeoPointActivity"
            android:theme="@style/Theme.Collect"
            tools:replace="android:theme" />
```

Reply:
The geo activities have a theme (`Theme.MaterialComponents`) set on them in their module manifest which allows them to function in isolation there for tests, so we override that there. For library activities we're basically overriding their "default" theme (hence the `tools:replace`).